host/sampi/http.go

```diff
@@ -95,23 +109,30 @@ func startJSONEventStreaming(w http.ResponseWriter) *json.Encoder {
 
 // HTTP Route Handles
 func (c *HTTPAPI) ListHosts(w http.ResponseWriter, r *http.Request, _ httprouter.Params) {
+	l := c.logger.New("fn", "ListHosts")
 	ret, err := c.Cluster.ListHosts()
 	if err != nil {
+		l.Error("error occured", "err", err)
```

Comment:
`s/error occured/error listing hosts`

I think it will be useful if I can understand the log message from the message itself, rather than needing to check the context too.